A GCC-specific setting from years ago. gcc-9 quietly ignores it (the
resulting binaries appear to be the same whether -fno-ipa-sra has been
specified or not), clang fails on it as not recognised.
Just get rid of it altogether.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,7 +100,7 @@
 if(APPLE)
    set(COMPILE_FLAGS "${COMPILE_FLAGS} -mmacosx-version-min=10.8 -stdlib=libc++ -Wno-error=c++11-narrowing -Wc++11-extensions -fpermissive")
 else()
-  set(COMPILE_FLAGS "${COMPILE_FLAGS} -Wno-error=narrowing -fno-ipa-sra -fpermissive")
+  set(COMPILE_FLAGS "${COMPILE_FLAGS} -Wno-error=narrowing -fpermissive")
 endif()
 
 if(NOT "${PRERELEASE_TAG}" STREQUAL "")
